app/test_main.py

```diff
@@ -1,3 +1,40 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (16, 16, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (28, 28, [3, 2]),
+        (-1, -1, [0, 0]),
+        (-35, -45, [0, 0])
```

Comment:
Add test cases to check against numbers that have different input values for `cat_age` and `dog_age` to make sure that values do not interfere with one another